clang/test/CheckedCRewriter/arrcalleemulti2.c

```diff
@@ -128,6 +128,6 @@ x = (int *) 5;
 z += 2;
 return z; }
 //CHECK_NOALL: int * sus(int *x, _Ptr<int> y) {
-//CHECK_NOALL:         int *z = calloc(5, sizeof(int)); 
-//CHECK_ALL: int * sus(int *x, _Ptr<int> y) {
-//CHECK_ALL:         int *z = calloc(5, sizeof(int)); 
```

Comment:
Any idea why the second `CHECK_NOALL` got removed? It would seem like it should be `int *` as before, right?

Reply:
Luckily, I know why this got removed. It's a specific bug that happens in multi-file tests only that has now been fixed :)
I didn't catch it because I slightly changed how the script searches for pointers to annotate and didn't port it over to multi-file rewriting.